packages/connect-react/src/index.ts

```diff
@@ -146,20 +157,36 @@ function useConnectSubscription<Data>(
       handler?.unsubscribe?.()
```

Comment:
Just wanted to check, when a component using a subscription is mounted, is it a new subscription each time? And the unsubs happen in the cleanup like here?

Reply:
> Just wanted to check, when a component using a subscription is mounted, is it a unique subscription?

Yes a new subscription is created for each component using a subscription hook once mounted.

> And the unsubs happen in the cleanup like here?

Yes exactly!